tests/unit/plugins/europeana/record.spec.js

```diff
@@ -0,0 +1,75 @@
+import nock from 'nock';
+import getRecord from '../../../../plugins/europeana/record';
+
+const axios = require('axios');
+axios.defaults.adapter = require('axios/lib/adapters/http');
+
+const apiUrl = 'https://api.europeana.eu';
+const apiEndpoint = '/api/v2/record/test/thisisatest.json';
+const apiKey = 'abcdef';
+const path = 'test/thisisatest';
+
+describe('plugins/europeana/record', () => {
+  describe('getRecord()', () => {
+
+    describe('when trying to request an unknown record', () => {
+      const errorMessage = 'Invalid record identifier';
+      beforeEach('stub API response', () => {
+        nock(apiUrl)
+          .get(apiEndpoint)
+          .query(true)
+          .reply(400, {
+            record: null,
+            error: errorMessage
+          });
+      });
+
+      it('returns API error message', () => {
+        const response = getRecord({
+          path: path,
+          key: apiKey
+        });
+        return response.should.eventually.have.property('error', errorMessage);
+      });
+    });
+
+    describe('when trying to request a record', () => {
+      const apiResponse = {
+        success: true,
+        object: {
+          aggregations: [{
+            edmIsShownAt: 'https://thisisalink.eu',
+            webResources: [{
+              about: 'https://thisistheabout.eu',
+              dcDescription: 'This is a description',
```

Comment:
A web resource's `dcDescription` is always a language map. Let's make it so here too.

e.g.
```json
"dcDescription": {
  "de": [
    "Umfang: F1 - f3",
    "Latcham, Michael: The pianos of Johann Andreas Stein. In: Zur Geschichte des Hammerklaviers. 14. Musikinstrumentenbau-Symposium in Michaelstein am 12. und 13. November 1993. Michaelstein 1996, S. 15-49."
  ]
}
```